### PR TITLE
feat(sbb-lead-container): initial implementation

### LINKED ISSUE #2668: story(lead-container): initial implementation (previously Head-Component)
We want to implement the `sbb-lead-container` component.
It's purpose is to be used at the top of a page with a lead image.

Figma: https://www.figma.com/file/9r6xSfNmEfCFxl1yFYedrj/Lyne-Components?type=design&node-id=35586-654830&mode=design&t=TyTftKcUUKiNAtkx-4

Design Spec:

- From zero to medium the image and content section (not the content itself) are both full width and the content follows the image below
- Starting from large, the content section has a specific width and is moved into the image space as depicted
- Starting from ultra, the image also has a fixed width

Tech Spec:

- It should have a slot for the image with name `image` (e.g. a `sbb-image`, `img` or `picture` can be slotted)
- It should have an unnamed slot for anything else which is placed inside the content section
- Add readme documentation for the spacings to be used inside the component (`4x` after `<sbb-breadcrumb-group>`, `4x` after a lead link below the `<sbb-breadcrumb-group>`, no spacing before the `<sbb-title>`, `s` after a lead text)

```html
<sbb-header>...</sbb-header>
<sbb-lead-container>
  <sbb-image slot="image" ...></sbb-image>

  <sbb-breadcrumb-group>...</sbb-breadcrumb-group>
  <sbb-block-link size="xs" icon-name="chevron-small-left-medium" icon-placement="start" href="...">Label</sbb-block-link>
  <sbb-title>Title</sbb-title>
  <p>Text</p>
  <div>Other content</div>
</sbb-lead-container>
```

### PR DESCRIPTION
**Pull request by jeripeierSBB**
Closes #2668 

Open topics:
 - @mcilurzo Is there a token for a new border-radius needed (currently set to fixed spacing 3x), or should it be another border-radius?
 - @mcilurzo How should I define the image overlapping distance, with "6rem"?
 - @mcilurzo Is the Story (example) an appropriate one, or do I need to change something?

### CODE REVIEW

**Comment by mcilurzo** [comment]
ux review: https://www.figma.com/design/9r6xSfNmEfCFxl1yFYedrj?node-id=38610-26669#814126529

**Comment by mcilurzo** [comment]
– Is there a token for a new border-radius needed (currently set to fixed spacing 3x), or should it be another border-radius?: i added a new token for the radius: https://www.figma.com/design/mWknI2rC5DJmOgRO61WKai/Lyne-Design-Tokens?node-id=1374-2656&t=m8nLjM0vMySuRkTa-11
– How should I define the image overlapping distance, with "6rem"?: with a fixed spacing (spacing-fixed-24x)?
– Is the Story (example) an appropriate one, or do I need to change something?: Perfect, no changes needed.

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Review by TomMenga — APPROVED** [review]
Just a little doubt
The rest il lgtm 👍

File: src/components/lead-container/lead-container.scss [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  // In case there is a global background color set, use full width with white color until large breakpoint
+  background-color: var(--sbb-lead-container-background-color);
+  @include sbb.mq($from: large) {
+    background-color: transparent;
+  }
+}
+
+.sbb-lead-container-content {
+  position: relative;
```
[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/components/lead-container/lead-container.scss:79]: Will re-check, but as far as I remember it was needed for stacking the picture to the content.

[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/components/lead-container/lead-container.scss:79]: was not needed, removed it (I thought that z-index always needs position relative...)

**Pull request merged**